deploy/olm-catalog/apicast-community-operator/apicast-community-operator.package.yaml

```diff
@@ -0,0 +1,7 @@
+packageName: apicast-community-operator
+channels:
+  - name: alpha
+    currentCSV: apicast-community-operator.v0.1.0
+  - name: threescale-2.7
```

Comment:
Food for though,
why not `alpha` and `stable` channels and publish apicast images based on upstream versioning??

Instead of `quay.io/3scale/3scale27:apicast-3scale-2.7.0-GA`
Use `quay.io/3scale/apicast:v3.7.0`

Reply:
wdyt @miguelsorianod :arrow_up_small: ?